authentication_sample.js

```diff
@@ -30,77 +29,50 @@ const secret = process.env['AZURE_CLIENT_SECRET'];
 // Random sample keyvault name
 const kvName = random_id();
 
-function authUsingAAD(vaultUri) {
-    console.log("Using AAD to authenticate to '" + vaultUri + "'");
-
-    const credentials = new ClientSecretCredential(tenantId, clientId, secret);
-    const secretClient = new SecretClient(vaultUri, credentials);
-    
-    // Using the key vault client, create and retrieve a sample secret.
-    console.log("Setting secret 'test-secret'");
-    
-    secretClient.setSecret('test-secret', 'test-secret-value')
-    .then( (kvSecretBundle) => {
-        console.log("Secret id: '" + kvSecretBundle.properties.id + "'.");
-
-        return secretClient.getSecret('test-secret');
-    })
-    .then( (bundle) => {
-        console.log("Successfully retrieved 'test-secret'");
-        console.log(bundle);
-    })
-    .catch( (err) => {
-        console.log(err);
-    });
-}
+const credentials = new ClientSecretCredential(tenantId,clientId,secret);
 
 // Sample setup: uses the resource management client to create a sample resource group
 // Then creates a key vault in this group and calls the authentication sample with the URI of the new vault. 
-function runSample(demoCallback) {
```

Comment:
This is not for credential purpose. it's a callback function. Please add it back.